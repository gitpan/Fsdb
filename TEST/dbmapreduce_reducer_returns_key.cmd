prog='dbmapreduce'
args='--no-prepend-key -k experiment -C "dbrow(\"--nolog\", \"_experiment eq 'ufs_mab_sys'\")"'
cmp='diff -cb '
in=TEST/dbmapreduce_ex.in
