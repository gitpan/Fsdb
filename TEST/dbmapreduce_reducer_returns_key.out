#fsdb experiment duration
ufs_mab_sys	37.2
ufs_mab_sys	37.3
# test trailing comment
#   | dbmapreduce --no-prepend-key -k experiment -C dbrow("--nolog", "_experiment eq 'ufs_mab_sys'")
